docroot/modules/custom/mass_entity_usage/tests/src/ExistingSite/EntityUsageTest.php

```diff
@@ -33,6 +33,11 @@ class EntityUsageTest extends ExistingSiteBase {
    */
   protected function setUp() {
     parent::setUp();
+
+    // Remove everything from the entity_usage table
+    // to avoid long cleaning times that break this test.
```

Comment:
Let's leave a hint for us in case we later have trouble with these events needing to be fired for future tests to work.

```suggestion
    // to avoid long cleaning times that break this test.
    // Note this avoids triggering events on bulk deletes, such as at
    // \Drupal\entity_usage\EntityUsage::bulkDeleteTargets().
```